lib/resolve-decl.js

```diff
@@ -143,18 +143,19 @@ function resolveDecl(decl, map, /*optional*/shouldPreserve, /*optional*/logResol
 	if(shouldPreserve === true && decl.value !== valueResults.value) {
 		decl.cloneAfter();
 	}
-
-
-	// Set 'undefined' value as a string to avoid making other plugins down the line unhappy
-	// See #22
-	if (valueResults.value === undefined) {
-		valueResults.value = 'undefined';
-	}
-
-
-	// Set the new value after we are done dealing with at-rule stuff
-	decl.value = valueResults.value;
-}
+
+
+	// Set 'undefined' value as a string to avoid making other plugins down the line unhappy
+	// See #22
+	if (valueResults.value === undefined) {
+		valueResults.value = 'undefined';
+	}
+
+
+	// Set the new value after we are done dealing with at-rule stuff
+	decl.value = valueResults.value;
+}
+

```

Comment:
Rebase this onto the current `master` and squash things down. This is already in the codebase